cli/src/ops/neon_new.ts

```diff
@@ -117,13 +141,39 @@ export default async function wizard(pwd: string, name: string) {
   answers.git = encodeURI(answers.git);
   answers.author = escapeQuotes(answers.author);
 
+  let neonVersion = await parseNeonVersion(neon);
+
+  let libs: {
+    // In the common case, we can make the Cargo.toml manifest simple by just using
+    // the semver specifier string for the `neon` and `neon-build` dependencies.
+    simple: boolean,
+    paths?: { neon: string, "neon-build": string },
+    version?: string,
+    features?: Array<string>
+  } = { simple: neonVersion.type !== 'path' && !features };
+
+  if (neonVersion.type === 'path') {
+    libs.paths = {
+      neon: path.resolve(neonVersion.value),
+      "neon-build": path.resolve(neonVersion.value, "crates", "neon-build")
```

Comment:
What do you think about using [`path.relative`](https://nodejs.org/api/path.html#path_path_relative_from_to) here? `path.resolve` will produce an absolute path which isn't very friendly to switching across environments (e.g., docker).

Reply:
How about we go with: it'll generate a relative path if you pass a relative path and an absolute path if you pass an absolute path.